CIAOS-Rust/client/ciaos/client.py

```diff
@@ -0,0 +1,238 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), 'util'))
+import requests
+from typing import List
+from util.flatbuffer.flatbuffer_handler import create_flatbuffer, parse_flatbuffer
+import config
+from typing import List, Optional
+
+class Ciaos:
+    def __init__(self):
+        """
+        Initialize Ciaos client with configuration parameters.
+        API URL and user ID are imported from config
+
+        config.API_URL = "<server url>"
+        config.user.id = "your_user_id"
+        """
+
+        # Get user ID from config
+        try:
+            self.user_id = config.user_id
+            if not self.user_id:
+                raise ValueError("User ID must be set in config.user.id")
+        except AttributeError:
+            raise ValueError("config.user.id must be defined")
+
+        self.headers = {
+            "User": self.user_id,
+        }
+
+    def put(self, file_path: str, key: Optional[str] = None):
```

Comment:
key should be the first parameter

Reply:
When you have a parameter with a default value, it should come after parameters without default values. If not, Python raises a SyntaxError or, as in this case, the linter (Pylance) flags it as an error.

Reply:
Umm But this is bad user experience.

Reply:
as you can see even boto3 follows same convention it has filename parameter before key 
[https://github.com/boto/boto3/blob/develop/boto3/s3/inject.py](https://github.com/boto/boto3/blob/develop/boto3/s3/inject.py)

Reply:
Let's go with this now. Address the below comment on config and we are good to go.